content/id/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -0,0 +1,639 @@
+1. (Opsional) Kecuali ditentukan sebelumnya, `kubeadm` akan menggunakan antarmuka jaringan yang diasosiasikan
+dengan _default gateway_ untuk mengatur alamat _advertise_ untuk peladen API pada simpul _control-plane_ ini.
+Untuk menggunakan antarmuka jaringan yang berbeda, tentukan argumen `--apiserver-advertise-address=<ip-address>` 
+pada `kubeadm init`. Untuk menyebarkan klaster Kubernetes IPv6 menggunakan pengalamatan IPv6, kamu
+harus menentukan alamat IPv6, sebagai contoh `--apiserver-advertise-address=fd00::101`
+1. (Opsional) Jalankan `kubeadm config images pull` sebelum `kubeadm init` untuk memastikan
+konektivitas ke registri _image_ kontainer gcr.io.
+
+Untuk menginisialisasi simpul _control-plane_ jalankan:
+
+```bash
+kubeadm init <args>
+```
+
+### Pertimbangan mengenai apiserver-advertise-address dan ControlPlaneEndpoint
+
+Meski `--apiserver-advertise-address` dapat digunakan untuk mengatur alamat _advertise_ untuk peladen
+API pada simpul _control-plane_ ini, `--control-plane-endpoint` dapat digunakan untuk mengatur _endpoint_ yang digunakan bersama
+untuk seluruh simpul _control-plane_.
+
+`--control-plane-endpoint` tidak hanya mengizinkan alamat IP tetapi juga nama DNS yang dapat dipetakan pada alamat IP.
+Silakan hubungi _network administrator_ untuk mengevaluasi solusi-solusi yang mempertimbangkan pemetaan tersebut.
+
+Berikut contoh pemetaannya:
+
+```
+192.168.0.102 cluster-endpoint
+```
+
+Di mana `192.168.0.102` merupakan alamat IP dari simpul ini dan `cluster-endpoint` merupakan nama DNS kustom yang dipetakan pada IP ini.
+Hal ini memungkinkan kamu untuk memberikan `--control-plane-endpoint=cluster-endpoint` pada `kubeadm init` dan memberikan nama DNS yang sama pada
+`kubeadm join`. Kemudian kamu dapat memodifikasi `cluster-endpoint` untuk mengarah pada alamat _load-balancer_ dalam skenario
+ketersediaan tinggi.
+
+Mengubah klaster _control plane_ tunggal yang dibuat tanpa `--control-plane-endpoint` menjadi klaster dengan ketersediaan tinggi
+tidak didukung oleh kubeadm.
+
+### Informasi lebih lanjut
+
+Untuk informasi lebih lanjut mengenai argumen-argumen `kubeadm init`, lihat [panduan referensi kubeadm](/docs/reference/setup-tools/kubeadm/kubeadm/).
+
+Untuk daftar pengaturan konfigurasi yang lengkap, lihat [dokumentasi berkas konfigurasi](/docs/reference/setup-tools/kubeadm/kubeadm-init/#config-file).
+
+Untuk menyesuaikan komponen-komponen _control plane_, termasuk pemasangan IPv6 opsional pada _liveness probe_ untuk komponen-komponen _control plane_ dan peladen etcd, berikan argumen ekstra pada tiap komponen seperti yang didokumentasikan pada [argumen-argumen kustom](/docs/setup/production-environment/tools/kubeadm/control-plane-flags/).
+
+Untuk menjalankan `kubeadm init` lagi, sebelumnya kamu harus [membongkar klaster](#pembongkaran).
+
+Jika kamu menggabungkan sebuah simpul dengan arsitektur yang berbeda ke klastermu, pastikan DaemonSets yang disebarkan
```

Comment:
```suggestion
Jika kamu menggabungkan sebuah Node dengan arsitektur (CPU) yang berbeda ke klastermu, pastikan DaemonSets yang di-_deploy_
```

Reply:
Resolved in https://github.com/kubernetes/website/pull/20945/commits/03745eaec90064d48ba7c612f74c7825115b8ffc